jasminewd2/index.d.ts

```diff
@@ -44,6 +44,13 @@ declare namespace jasmine {
     toThrowError(expected?: new (...args: any[]) => Error | Promise<new (...args: any[]) => Error>, message?: string | RegExp | Promise<string | RegExp>): Promise<void>;
   }
 
+  interface ArrayLikeMatchers<T extends ArrayLike<T>> extends Matchers<T> {
```

Comment:
The code would still look like this with regular arrays though

```
interface ArrayMatchers<T extends Array<T>> extends Matchers<T> {
```

As I want to constrain the type `T` to be compatible with `Array<T>` or is there some error I don`t see?